src/library/Uno.Material/Styles/Controls/v2/ToggleButton.xaml

```diff
@@ -0,0 +1,306 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:um="using:Uno.Material"
+					xmlns:xamarin="http://uno.ui/xamarin"
+					mc:Ignorable="d xamarin">
+
+	<!-- ToggleButton Brushes -->
+	<SolidColorBrush x:Key="M3MaterialToggleButtonTextLabelBrush"
+					 Color="{ThemeResource PrimaryColor}" />
+
+	<StaticResource x:Key="M3MaterialToggleButtonForegroundThemeBrush"
+					ResourceKey="OnSurfaceBrush" />
+
+	<!-- Other ToggleButton Variables -->
+	<CornerRadius x:Key="M3MaterialToggleButtonBorderRadius">4</CornerRadius>
+	<x:Double x:Key="M3MaterialToggleButtonFontSize">14</x:Double>
+	<Thickness x:Key="M3MaterialToggleButtonPadding">16,8</Thickness>
+
+	<Style x:Key="M3MaterialTextToggleButtonStyle"
+		   TargetType="ToggleButton">
+		<Setter Property="Foreground"
+				Value="{StaticResource M3MaterialToggleButtonTextLabelBrush}" />
+		<Setter Property="Background"
+				Value="Transparent" />
+		<Setter Property="BorderThickness"
+				Value="0" />
+		<Setter Property="Padding"
+				Value="{ThemeResource M3MaterialToggleButtonPadding}" />
+		<Setter Property="CornerRadius"
+				Value="{StaticResource M3MaterialToggleButtonBorderRadius}" />
+		<Setter Property="MinHeight"
+				Value="40" />
+		<Setter Property="FontFamily"
+				Value="{ThemeResource MaterialRegularFontFamily}" />
+		<Setter Property="FontSize"
+				Value="{ThemeResource M3MaterialToggleButtonFontSize}" />
+		<Setter Property="UseSystemFocusVisuals"
+				Value="True" />
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="ToggleButton">
+					<Grid x:Name="Root"
+						  Background="{TemplateBinding Background}"
+						  BorderBrush="{TemplateBinding BorderBrush}"
+						  BorderThickness="{TemplateBinding BorderThickness}"
+						  CornerRadius="{TemplateBinding CornerRadius}">
+						<VisualStateManager.VisualStateGroups>
+							<VisualStateGroup x:Name="CommonStates">
+								<VisualState x:Name="Normal">
+									<VisualState.Setters>
+										<Setter Target="HoverOverlay.Opacity"
+												Value="0" />
+										<Setter Target="FocusedOverlay.Opacity"
+												Value="0" />
+									</VisualState.Setters>
+								</VisualState>
+								<VisualState x:Name="PointerOver">
+									<Storyboard>
+										<DoubleAnimation Storyboard.TargetName="HoverOverlay"
+														 Storyboard.TargetProperty="Opacity"
+														 Duration="{StaticResource MaterialAnimationDuration}"
```

Comment:
Just curious to know if MaterialAnimationDuration changed in a way between M2 and M3 ?

Reply:
I don't think so